### PR TITLE
Create ui-c8y-1021-5-1-ui-c-8-y-properties-list-the-placement-of-no-parse.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59862](https://cumulocity.atlassian.net/browse/MTM-59862)
Original PR: [7510](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7510)

[MTM-59862]: https://cumulocity.atlassian.net/browse/MTM-59862?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-5-1-ui-c-8-y-properties-list-the-placement-of-no-parse.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: UI c8y properties list the placement of no parse should not matter (#6808) (#7510)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-5-1-ui-c-8-y-properties-list-the-placement-of-no-parse.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved handling of noParse attribute in c8y-properties-list UI component
```

File: content/change-logs/application-enablement/ui-c8y-1021-5-1-ui-c-8-y-properties-list-the-placement-of-no-parse.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59862
+version: 1021.5.1
+---
+UI c8y properties list the placement of no parse should not matter (#6808) (#7510)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-5-1-ui-c-8-y-properties-list-the-placement-of-no-parse.md:17]: The Claude AI suggested to use the following body:
```suggestion
The c8y-properties-list UI component is used to display a list of properties in Cumulocity IoT applications. Previously, the placement of the noParse attribute within the component's configuration could lead to inconsistent behavior. With this change, the handling of the noParse attribute has been improved, ensuring that its placement within the configuration no longer affects the component's functionality. This enhancement provides a more reliable and predictable experience for users when utilizing the c8y-properties-list component in their applications.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by eniosultan** [comment]
PR closed due to the task not being relevant for users...

**Pull request closed without merging**